src/mds/Server.cc

```diff
@@ -580,6 +580,13 @@ void Server::handle_client_session(const cref_t<MClientSession> &m)
   uint64_t sseq = 0;
   switch (m->get_op()) {
   case CEPH_SESSION_REQUEST_OPEN:
+    if(mds->mdsmap->get_reject_client_session()) {
+      dout(0) << "new sessions are not permitted, enable again via `ceph fs ...`" << dendl;
```

Comment:
```suggestion
      dout(0) << "new sessions are not permitted, enable again via `ceph fs ...`" << dendl;
```

Please fill in `ceph fs set ...`

Reply:
done